ovirtlib4/vms.py

```diff
@@ -63,6 +63,38 @@ def disks(self):
     def backups(self):
         return VmBackups(connection=self.service)
 
+    def start_and_wait(self, state='up'):
+        """
+        Start VM and wait for it to start
+
+        Args:
+            state (str): VM state to wait for (default 'up')
+
+        Returns:
+             VmEntity: The updated VM object
```

Comment:
maybe return true or false if start succeeded

Reply:
It better to return the object, in this case, you can see the status and validate it is as expected and also there are
much more VM parameters, that you can track
 e.g.:
vm = engine.vms()[0]
vm.start_and_wait().entity.status

Reply:
so you can change the method name to:
 vm.start_and_wait_and_get().entity.status :)

Reply:
without this method you would do:

vm = engine.vms()[0]
vm.service.start()
vm.get(wait_for="entity.status.value='up'", wait_timeout=60) 
 # or you can also omit the get() and call it like that
vm(wait_for="entity.status.value='up'", wait_timeout=60)  # this will return list of objects, currently only one

So actually the "wait" operation is a get() request not need to mention in the method name
This method is just a helper for this common operation

Reply:
but it should return None on failure, so I will add it to the docstring, thx:)